packages/scrollable-video/src/components/video.js

```diff
@@ -93,6 +93,34 @@ function removeListenerFromMQList(mql, listener) {
   }
 }
 
+/**
+ *
+ * On iOS low power mode, autoplaying video is not allowed.
+ * Also, video cannot be played unless user interact with the device in such situation.
+ * This hack plays video on user first touch for hiding the playback button UI when it cannot be autoplayed.
+ */
+function playVideoWhenAutoplayIsDisabledOnTouch() {
+  if (typeof window === 'undefined') return
+  const playVideoWhenAutoplayIsDisabled = () => {
+    const videos = document.querySelectorAll('video')
+    if (videos && videos.length > 0) {
+      videos.forEach(video => {
+        if (!video.playing) {
+          // video should be autoplayed.
+          // if video is not playing, play video now.
+          const playPromise = video.play()
```

Comment:
If the video is not in the viewport, will the video be played as well?
If the video not in the viewport is played, will this cause side effects?

Or the video will be stopped immediately due to `onPlay` event?

Reply:
Yes, the video will be paused immediately due to `onPlay` event?